### ISSUE TITLE
iOS Issue - pytorch 1.60

### ISSUE THREAD

**Issue by zy885137** [pytorch/pytorch#42304]
Unknown builtin op: aten::mul.
Could not find any similar ops to aten::mul. This op may not exist or may not be currently supported in TorchScript.
:
  File "<string>", line 3

def mul(a : float, b : Tensor) -> Tensor:
  return b * a
         ~~~~~ <--- HERE
def add(a : float, b : Tensor) -> Tensor:
  return b + a

**Comment by xta0** [comment]
Hi @zy885137. Do you mind sharing your steps so that I can repro?

**Comment by AtomLee4279** [comment]
I met this issue too. Did you solve it? @zy885137

**Comment by zy885137** [comment]
> I met this issue too. Did you solve it? @zy885137
1. You can see if this problem can be solved first: https://github.com/pytorch/pytorch/issues/42361
2. Rename the math.h file of Libtorch

**Comment by AtomLee4279** [comment]
But I did not meet your problem like  #42361.I did not get  logs like "PyTorch is not linked with support for cpu devices (getDeviceGuardImpl at /Users/distiller/project/c10/core/impl/DeviceGuardImplInterface.h:216)
(no backtrace available)"
 I compiled pytorch 1.8.0 source code for ios by myself. (https://pytorch.org/tutorials/prototype/ios_gpu_workflow.html?highlight=from_blob)
then , I tried to call "torch::jit::load(filePath.UTF8String)" and finally I got the error message ："Unknown builtin op: aten::mul.
Could not find any similar ops to aten::mul. This op may not exist or may not be currently supported in TorchScript.
:
  File "<string>", line 3

def mul(a : float, b : Tensor) -> Tensor:
  return b * a
         ~~~~~ <--- HERE
def add(a : float, b : Tensor) -> Tensor:
  return b + a"，just like you.

**Comment by AtomLee4279** [comment]
> > I met this issue too. Did you solve it? @zy885137
> 
> 1. You can see if this problem can be solved first: #42361
> 2. Rename the math.h file of Libtorch

@zy885137 
But I didn't  meet your problem like #42361 ,also I did not get any message like "PyTorch is not linked with support for cpu devices (getDeviceGuardImpl at /Users/distiller/project/c10/core/impl/DeviceGuardImplInterface.h:216)
(no backtrace available)".
By the way , I compiled pytorch 1.8.0 source code for IOS (https://pytorch.org/tutorials/prototype/ios_gpu_workflow.html?highlight=from_blob).
Then I tried to called "torch::jit::load(filePath.UTF8String)" and finally I got error logs 
"Unknown builtin op: aten::mul.
Could not find any similar ops to aten::mul. This op may not exist or may not be currently supported in TorchScript.
:
  File "<string>", line 3

def mul(a : float, b : Tensor) -> Tensor:
  return b * a
         ~~~~~ <--- HERE
def add(a : float, b : Tensor) -> Tensor:
  return b + a" ,just like you.
